fv/utils/utils.go

```diff
@@ -86,14 +82,16 @@ func run(input []byte, checkNoError bool, command string, args ...string) error
 		cmd.Stdin = bytes.NewReader(input)
 	}
 	outputBytes, err := cmd.CombinedOutput()
-	currentTestOutput = append(currentTestOutput, fmt.Sprintf("Command: %v %v\n", command, args))
-	currentTestOutput = append(currentTestOutput, string(outputBytes))
+	output := string(outputBytes)
 	LastRunOutput = string(outputBytes)
 	if err != nil {
 		log.WithFields(log.Fields{
 			"command": command,
-			"args":    args,
-			"output":  string(outputBytes)}).WithError(err).Warning("Command failed")
+			"args":    args}).WithError(err).Warningf("Command failed:\n%s", output)
+	} else {
+		log.WithFields(log.Fields{
+			"command": command,
+			"args":    args}).Infof("Command succeeded:\n%s", output)
```

Comment:
I think it's going to be a bit confusing having ` command=kubectl whatever` appended to the last line of the output.  Can we revert to logging the command on its own first?  It's not a big deal for me, but I think that would be better.

Reply:
Yes, was just thinking the same myself after trying this out for a bit :+1: